sql/src/main/java/io/crate/expression/tablefunctions/TableFunctionModule.java

```diff
@@ -32,5 +32,6 @@ public void configureFunctions() {
         UnnestFunction.register(this);
         EmptyRowTableFunction.register(this);
         GenerateSeries.register(this);
+        ValuesFunction.register(this);
```

Comment:
is it ok to register it, but not expose/document? Otherwise, we also can create an instance of the function in the `RelationAnalyzer#visitValues` without registering it in functions.